plugins/serializers/extr/README.md

```diff
@@ -2,14 +2,19 @@
 
 The `extr` output data format converts metrics into JSON documents, performing the following operatins on batched metrics:
    - Combines sequential metrics matching name, tags, and timestamps into a single JSON metric, combining the fields of each metric into an array of fields.
-   - Groups metric fields appended with _min, _max, _avg
+   - Groups metric fields appended with _min, _max, _avg or _old, _new
         usage_min=1,usage_max=100,usage_avg-50
         --> "usage":{"avg":50,"max":100,"min":1}
+        ifAdminStatus_old=1,ifAdminStatus_new=1
+        --> "ifAdminStatus":{"old":1,"new":0}
    - Groups metric fields appended with _key.
-       ifIndex_key=1, name_key="1:2"
-       --> "key":{ifIndex:1, name:"1:2:}
+        ifIndex_key=1, name_key="1:2"
+        --> "key":{ifIndex:1, name:"1:2:}
+   - Groups metric fields appended with _tag.
+        partNumber_tag="1647G-00129 800751-00-01", revision_tag="01"
+        --> "tags":{ifIndex:"1647G-00129 800751-00-01", revision:"01"}
```

Comment:
ifIndex should be partNumber_tag on this line?

Reply:
I'll fix